### ISSUE TITLE
Popover: Exceptions / Unable to Close by Clicking Outside

### ISSUE THREAD

**Issue by hoverduck** [Automattic/wp-calypso#90]
While testing the editor in IE11, I've encountered some issues with the `<Popover />` component and not being able to close by clicking outside.  The following error is thrown when interacting with various Popovers, like the Post Visibility Popover.

![image](https://cloud.githubusercontent.com/assets/7233112/11245750/442df118-8de2-11e5-8c6d-0beecb039af8.png)

This results in only being able to close the popover by clicking on the parent/context element.

**Steps to Reproduce**
- Open the editor in IE11
- Click on the Post Visibility Icon
- Note errors in the console
- Click outside of the popover, note that dialog does not close

Node.contains() is supported from IE5.0: https://developer.mozilla.org/en-US/docs/Web/API/Node/contains

However there is a side note on that page: [only supports contains() for elements.](https://connect.microsoft.com/IE/feedback/details/780874/node-contains-is-incorrect). This page has many interesting comments about this issue.

This is strange, because `svg` has `nodeType` 1 (`ELEMENT_NODE`) but also does not support `.contains()`.  Also Internet Explorer returns the `nodeName` for my `svg` element in lower-case, because [why not](http://ejohn.org/blog/nodename-case-sensitivity/).
